src/style.css

```
@@ -0,0 +1,100 @@
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 350;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: bold;
+  font-weight: 850;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  margin-top: 0;
+  padding-top: 20px;
+  font-family: Avenir, sans-serif;
+}
+
+.form {
+  padding: 0;
+  margin: 0;
+  border: none;
+}
+
+.label {
+  padding: 0;
+  margin: 0;
+  border: none;
+}
+
+.input,
+.inputSmall {
+  width: 100%;
+  margin-top: 0;
+  margin-bottom: 20px;
+  height: 70px;
+  padding-top: 25px;
+  padding-bottom: 23px;
+  padding-left: 62px;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 100%;
+  color: #3d4e61;
+  font-style: normal;
+  font-weight: 350;
+  background: #fff url("./images/Search.svg") no-repeat;
+  background-position: left 26px center, left top;
+  background-size: 19px, 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.inputSmall {
+  height: 42px;
+  padding-top: 9px;
+  padding-bottom: 10px;
+  padding-left: 33px;
+  font-size: 14px;
+  background-position: left 13px center, left top;
+  background-size: 11px, 100%;
+}
+
+.input:hover,
+.inputSmall:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus,
+.inputSmall:focus {
+  background-image: url("./images/Search.svg"), linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-position: left 26px center, left top;
+  background-size: 19px, 100%;
+  outline: none;
+  box-shadow: none;
+  border-radius: 3px;
+  font-style: bold;
+  font-weight: 850;
+}
+
+.inputSmall:focus {
+  background-position: left 13px center, left top;
+  background-size: 11px, 100%;
+}
+
+.input::placeholder,
+.inputSmall::placeholder {
+  font-family: Avenir, sans-serif;
+  line-height: 100%;
+  color: #3d4e61;
+  font-style: normal;
+  font-weight: 350;
+  font-size: 16px;
+}
```

```suggestion
.input::placeholder {
  font-family: Avenir, sans-serif;
  line-height: 100%;
  color: #3d4e61;
  font-style: normal;
  font-weight: 350;
  font-size: 16px;
}
```
you don't need it - just add 2 classes on element